src/main/java/Duke.java

```diff
@@ -1,10 +1,365 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.ArrayList;
+
+/**
+ * An interactive bot that performs various tasks based on user command
+ * <p>
+ * Last updated : 15 September 2020
+ * <p>
+ * Functions implemented:
+ * 1) Adding tasks to a list
+ * 2) Printing the list of tasks stored
+ * 3) Indicating which task is done
+ * 4) Different type of tasks (event, deadline, todo)
+ * 5) Delete tasks
+ * 6) Task storage onto a txt file
+ */
+
 public class Duke {
+    /**
+     * Main function
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        ArrayList<Task> listOfTasks = new ArrayList<>();
+        // Load the storage file content
+        try {
+            DukeFiles.initializeFile(listOfTasks);
+        } catch (IOException e) {
+            System.out.println("Oh no! There's a problem loading file content");
+        }
+        greet();
+        processCommand(listOfTasks);
+        exit();
+    }
+
+    /**
+     * Prints greet user message and a simple user guide
+     */
+    public static void greet() {
+        printDivider();
+        System.out.println("Hey mate! Nice to meet you, I'm Duke!\nHow can I help you?\n");
+        System.out.println("Here are some commands you can use to interact with me:\n" +
+                "todo <task>                : Store what needs to be done \n" +
+                "deadline <task> /by <date> : Keep track of your deadlines!\n" +
+                "event <task> /at <date>    : To keep in mind upcoming important events!\n" +
+                "list                       : To list out all the tasks you have so far\n" +
+                "done <integer number>      : To mark a task as done\n" +
+                "delete <integer number>    : To delete a task from the list\n");
+        System.out.println("Go ahead!");
+        printDivider();
+    }
+
+    /**
+     * Prints goodbye message
```

Comment:
You are printing a goodbye message, why your method is called "exit()". Maybe you should double-ckeck the method's name to make it more informative :+1:

Reply:
Thanks! Will update it